tests/db/conftest.py

```diff
@@ -98,7 +98,7 @@ def fin():
 
 
 @pytest.fixture
-def inputs(user_vk, amount=1, b=None):
+def inputs(user_vk, amount=4, b=None, blocks=4):
```

Comment:
Good catch!
I think the parameters were there because the fixture before was a function.